### PR TITLE
Added exclusion criteria, for cases where VCF CHROM is not recognized.

### LINKED ISSUE #19: Graceful handling of unknown chromosome
Currently, when converter encounters an unknown chromosome, it is unable to determine a corresponding reference sequence, resulting in an exception, as in this case: 

<pre>#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	Sample
chrR	301	.	A	G	.	.	.	GT:PS	0/1:.</pre>

We will likely want to add another exclusion criteria, for cases where VCF CHROM is not recognized. These would then also go into the invalidRecord log.

### PR DESCRIPTION
**Pull request by Rohan-cod**
This commit fixes #19.

### CODE REVIEW

**Review by srgothi92 — COMMENTED** [review]
Thanks for this contribution! We are happy to get more and more external contributors. 

I have suggested some changes we should make before merging this.

**Comment by Rohan-cod** [comment]
Thank you for the review Sir. I'll make all the changes as suggested by you.

**Comment by Rohan-cod** [comment]
@srgothi92, I have made all the changes as suggested by you. Please review them.

File: vcf2fhir/test/test_vcf2fhir.py [hunk 1]
```diff
@@ -413,6 +413,15 @@ def test_chrom_M(self):
                 chrom), expected_chrom[i])
             i += 1
 
+    def test_chrom_validation(self):
+        actual_chrom = ['chrX', 'R', 'mt', 'chrR', 'chrM', 'chrMT', 'chr30', 'P', 'Z', '45', 'o']
```
Comment (srgothi92) [inline vcf2fhir/test/test_vcf2fhir.py:417]: Nice. Can we add few more valid cases to tests mainly `CHRX`, `1`, `22`

**Review by srgothi92 — COMMENTED** [review]
Changes looks good , Nice work !

Can we squash both commits and also add some details to commit message. At least one line for about what this commit would fix. Issue number linking are good for additional info but you still need proper commit messages.

**Comment by Rohan-cod** [comment]
@srgothi92, I added more valid cases to the test function, squashed the commits into a single commit, and updated its description. Please review.

**Review by srgothi92 — APPROVED** [review]
Nice 🎉

**Pull request merged**